ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -1895,3 +1895,60 @@ def test028_update_test_unit_with_delete_in_form(self):
         testunit_name = row_with_headers['Test Unit']
         self.base_selenium.LOGGER.info(" + Test unit : {}".format(testunit_name))
         self.assertIn(testunit_name, testunit_name)
+
+    def test029_update_material_type(self):
+        """
+        Apply this on suborder number 5 for example:-
+        -When user update the materiel type from table view once I delete it message will appear
+        (All analysis created with this order and test plan will be deleted )
+        -Once you press on OK button, the material type & article & test pan will delete
+        -You can update it by choose another one and choose corresponding article & test plan 
+
+        LIMS-4264
+        
+        """
+        # Get test plan
+        test_plan_dict = self.get_active_article_with_tst_plan(
+            test_plan_status='complete')
+        
+        # Get random order
+        order_request = self.orders_api.get_all_orders(limit=20).json()
+        self.assertEqual(order_request['status'], 1)
+        orders_records = order_request['orders']
+        self.assertNotEqual(len(orders_records), 0)
+        random_order_index = self.generate_random_number(lower=0, upper=len(orders_records) - 1)
+        selected_order_record = orders_records[random_order_index]
+
+        self.order_page.get_orders_page()
+        order_page_url = self.base_selenium.get_url()
+        edit_page_url = order_page_url+'/'+str(selected_order_record['id'])
+        self.base_selenium.get(edit_page_url)
```

Comment:
why you need those two lines & you need the sub order index that you will update the material type & test plan & article for it